.github/workflows/linux.yml

```
@@ -35,14 +35,15 @@ jobs:
     container:
       image: 'openvinogithubactions.azurecr.io/openvino_provider:0.1.0'
       volumes: [ "/mount:/mount" ]
```

```suggestion
      volumes: [ "/mount:/mount", "${{ github.workspace }}:${{ github.workspace }}" ]
```
Could you check if that works